data_steward/retraction/retract_deactivated_pids.py

```diff
@@ -391,12 +389,15 @@ def create_queries(project_id, ticket_number, pids_project_id, pids_dataset_id,
                         date_column=date_row.date_column)
                 queries_list.append({
                     clean_consts.QUERY: sandbox_query,
-                    clean_consts.DESTINATION_DATASET: sandbox_dataset,
+                    clean_consts.DESTINATION: date_row.project_id + '.' + date_row.dataset_id + '.' + date_row.table,
```

Comment:
I think you need to change the `dataset_id` here.  Download this [patch.txt](https://github.com/all-of-us/curation/files/4726263/patch.txt) and apply it.  You can see it is saving the sandboxed rows to the same table as the cleaning queries.  I really don't know how it is doing it.  I was expecting to see sandboxed tables in `lbiggers_ehr_sandbox` but that dataset is empty.  If I'm misinterpreting something, let me know.
```
# Download patch.
# Do the following from  the command line.
# assumes the patch is downloaded to your ~/Downloads folder
# move the patch to your working directory
$ mv ~/Downloads/patch.txt "<path to>/curation/data_steward"

# move to your working directory if you're not already there
$ cd "<path to>/curation/data_steward"

# confirm patch.txt is in your directory
$ ls 

# checkout branch the patch is based on.
$ git checkout am/retract_deactivated_ehr

# apply the patch.    if there have been changes since 2020/06/03 3:00 PM, there may be conflicts.  if there are, I will provide a new patch file.
$ git apply < patch.txt

# verify the patch took
$ git diff

# this should show a change to the integration test as well as some print statements in the actual module
# move up a directory, ensure your environment is sourced, and run the integration test.
$ cd ..
$ ./tests/run_tests.sh -s integration -r retract_deactivated_pids*
```

Reply:
@lrwb-aou This code that you're commenting on is outdated. I downloaded the patch and ran the integration test-- below are my print statements, I also paused the run to verify the sandbox tables were in the correct dataset in BigQuery and they are showing correctly on my end, I have a screenshot below as well. Let me know what you're seeing different?

```
sandbox destination is:  			aou-res-curation-test.amcgrenera_ehr_sandbox.DCXXX_fake_condition_occurrence
OMOP table destination is:  			aou-res-curation-test.amcgrenera_ehr.fake_condition_occurrence
sandbox destination is:  			aou-res-curation-test.amcgrenera_ehr_sandbox.DCXXX_fake_drug_exposure
OMOP table destination is:  			aou-res-curation-test.amcgrenera_ehr.fake_drug_exposure
sandbox destination is:  			aou-res-curation-test.amcgrenera_ehr_sandbox.DCXXX_fake_measurement
OMOP table destination is:  			aou-res-curation-test.amcgrenera_ehr.fake_measurement
sandbox destination is:  			aou-res-curation-test.amcgrenera_ehr_sandbox.DCXXX_fake_observation
OMOP table destination is:  			aou-res-curation-test.amcgrenera_ehr.fake_observation
sandbox destination is:  			aou-res-curation-test.amcgrenera_ehr_sandbox.DCXXX_fake_procedure_occurrence
OMOP table destination is:  			aou-res-curation-test.amcgrenera_ehr.fake_procedure_occurrence
sandbox destination is:  			aou-res-curation-test.amcgrenera_ehr_sandbox.DCXXX_fake_visit_occurrence
OMOP table destination is:  			aou-res-curation-test.amcgrenera_ehr.fake_visit_occurrence
sandbox destination is:  			aou-res-curation-test.amcgrenera_ehr_sandbox.DCXXX_fake_condition_occurrence
OMOP table destination is:  			aou-res-curation-test.amcgrenera_ehr.fake_condition_occurrence
sandbox destination is:  			aou-res-curation-test.amcgrenera_ehr_sandbox.DCXXX_fake_drug_exposure
OMOP table destination is:  			aou-res-curation-test.amcgrenera_ehr.fake_drug_exposure
sandbox destination is:  			aou-res-curation-test.amcgrenera_ehr_sandbox.DCXXX_fake_measurement
OMOP table destination is:  			aou-res-curation-test.amcgrenera_ehr.fake_measurement
sandbox destination is:  			aou-res-curation-test.amcgrenera_ehr_sandbox.DCXXX_fake_observation
OMOP table destination is:  			aou-res-curation-test.amcgrenera_ehr.fake_observation
sandbox destination is:  			aou-res-curation-test.amcgrenera_ehr_sandbox.DCXXX_fake_procedure_occurrence
OMOP table destination is:  			aou-res-curation-test.amcgrenera_ehr.fake_procedure_occurrence
sandbox destination is:  			aou-res-curation-test.amcgrenera_ehr_sandbox.DCXXX_fake_visit_occurrence
OMOP table destination is:  			aou-res-curation-test.amcgrenera_ehr.fake_visit_occurrence
sandbox destination is:  			aou-res-curation-test.amcgrenera_ehr_sandbox.DCXXX_fake_condition_occurrence
OMOP table destination is:  			aou-res-curation-test.amcgrenera_ehr.fake_condition_occurrence
sandbox destination is:  			aou-res-curation-test.amcgrenera_ehr_sandbox.DCXXX_fake_drug_exposure
OMOP table destination is:  			aou-res-curation-test.amcgrenera_ehr.fake_drug_exposure
sandbox destination is:  			aou-res-curation-test.amcgrenera_ehr_sandbox.DCXXX_fake_measurement
OMOP table destination is:  			aou-res-curation-test.amcgrenera_ehr.fake_measurement
sandbox destination is:  			aou-res-curation-test.amcgrenera_ehr_sandbox.DCXXX_fake_observation
OMOP table destination is:  			aou-res-curation-test.amcgrenera_ehr.fake_observation
sandbox destination is:  			aou-res-curation-test.amcgrenera_ehr_sandbox.DCXXX_fake_procedure_occurrence
OMOP table destination is:  			aou-res-curation-test.amcgrenera_ehr.fake_procedure_occurrence
sandbox destination is:  			aou-res-curation-test.amcgrenera_ehr_sandbox.DCXXX_fake_visit_occurrence
OMOP table destination is:  			aou-res-curation-test.amcgrenera_ehr.fake_visit_occurrence
```

![image](https://user-images.githubusercontent.com/56938070/83693188-3121fd80-a5bb-11ea-92b8-c23842d05dcd.png)

Reply:
This was my fault.  I forgot to fetch the new changes and just did a checkout.  This looks good and correct.